tests/config/nightwatch.conf.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable */
+const argv = require('yargs')
+  .usage('Usage: $0 --target=[string]')
+  .argv;
+require('babel-core/register')({
+  only: [/app/, /tests/, /components/]
+});
+// require("babel-polyfill");
```

Comment:
not sure which line you're talking about, but the target is useful for 'pre-prod' testing when we get to continuous deployment i.e. you set the target and the e2e tests will run a different url.